### ISSUE TITLE
[ShapeInferenceError] Shape inference error(s): (op_type:Unsqueeze, node name: Unsqueeze__245)

### ISSUE THREAD

**Issue by bdnkth** [onnx/tensorflow-onnx#1456]
**Describe the bug**
Converting the CenterNet Resnet50 V1 FPN 512x512 from the TF Object Model zoo after it has been trained on custom data results in the following error:
`[ShapeInferenceError] Shape inference error(s): (op_type:Unsqueeze, node name: Unsqueeze__245): [ShapeInferenceError] values in 'axes' are beyond the bounds of the computed output shape`

The model has been trained with TF 2.4.1 for some epochs on my custom data and has been exported with the Object Detection API 0.1, which is the newest version.
The exported onnx model runs through the onnx.checker.check_model() but if it is loaded into the onnxruntime it fails with `onnxruntime::HandleNegativeAxis axis >= -tensor_rank && axis <= tensor_rank - 1 was false. axis 1 is not in valid range [-1,0]`

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Tensorflow Version: 2.4.1
- Python version: 3.8.8
- TF2onnx version: newest master

**To Reproduce**
The TF saved model can be found [here](https://we.tl/t-SHRarjK4My). Export it with tf2onnx the error messages come during the convert process.

**Comment by TomWildenhain-Microsoft** [comment]
Sorry, can you upload the file again?  Thanks.

**Comment by bdnkth** [comment]
I have reuploaded the [saved model](https://we.tl/t-Yv0XFB3yG2).

**Comment by TomWildenhain-Microsoft** [comment]
Yep, we had a bug in Slice conversion.
